src/style/_misc.scss

```diff
@@ -19,3 +19,14 @@ $z-index-tooltip: 5;
     box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
   }
 }
+
+@mixin hideText {
```

Comment:
Was thinking this should maybe be something more generic, as you could be hiding something other than text?

I had it as `@mixin hidden`. No biggie.

Reply:
Good idea, I've updated it to match yours.